docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -859,111 +859,86 @@ Administrators can increase this if they experience OAuth login state mismatch e
 
 > **Note**: This option is deprecated in favor of OAuth provider specific `skip_org_role_sync` settings. The following sections explain settings for each provider.
 
-Skip forced assignment of OrgID `1` or `auto_assign_org_id` for external logins. Default is `false`.
-Use this setting to allow users with external login to be manually assigned to multiple organizations.
-
-By default, the users' organization and role is reset on every new login.
-
 > **Warning**: Currently if no organization role mapping is found for a user, Grafana doesn't update the user's organization role.
 > With Grafana 10, if `oauth_skip_org_role_update_sync` option is set to `false`, users with no mapping will be
 > reset to the default organization role on every login. [See `auto_assign_org_role` option]({{< relref ".#auto_assign_org_role" >}}).
 
 ### skip_org_role_sync
 
-To prevent synchronization of organization roles for a specific OAuth integration, you can set the `skip_org_role_sync` option to `true`.
-This option is useful when you want to manage the organization roles of your users from within Grafana or when you want to prevent synchronization conflicts when they are synchronized from another provider.
-
-### [auth.grafana_com] skip_org_role_sync
-
-To prevent synchronization of organization roles for a specific OAuth integration, you can set the `skip_org_role_sync` option to `true`. Please note that there is a deprecated setting called `oauth_skip_org_role_update_sync` which has a different scope. While `skip_org_role_sync` only applies to the specific OAuth provider, `oauth_skip_org_role_update_sync` is a generic setting that affects all configured OAuth providers.
-
-The table below show the OAuth provider and their setting with the default value and the skip org role sync setting.
-| OAuth Provider | `oauth_skip_org_role_sync_update` | `skip_org_role_sync` | Behavior |
-| --- | --- | --- | --- |
-| Grafana.com | false | false | will sync with Grafana.com roles |
-| Grafana.com | true | false | skip org role sync for OAuth providers including Grafana.com users |
-| Grafana.com | false | true | skip org role sync for grafana.com users |
-| Grafana.com | true | true | skip org role sync for Grafana.com users and all other OAuth providers |
-
-### [auth.azuread] skip_org_role_sync
+If you want to manage the organization roles of your users from within Grafana or when you want to prevent synchronization conflicts when they are synchronized from another provider.
```

Comment:
```suggestion
Use this setting when you want to manage the organization roles of your users from within Grafana or to prevent synchronization conflicts when they are synchronized from another provider.
```

Reply:
@eleijonmarck , Just checking on wording...is this a 'setting' or a 'parameter'?